common/audio/audio_formats.cpp

```diff
@@ -289,8 +289,7 @@ void test_encode_adpcm(const std::vector<s16>& samples,
       for (int i = 0; i < 5; i++) {
         fmt::print(" [{}] {} {}\n", i, filter_errors[i], filter_shifts[i]);
       }
-      fmt::print("prev: {} {}\n", prev_block_samples[0], prev_block_samples[1]);
-      ASSERT(false);
+      ASSERT_MSG(false, fmt::format("prev: {} {}", prev_block_samples[0], prev_block_samples[1]));
```

Comment:
if we wanted to be really fancy, there could be an `ASSERT_FMT` that does the call to `fmt::format`. But writing that macro is hard...